recipes/match-series/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "match-series" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/berkels/{{ name }}/archive/master.tar.gz
+  sha256: 51982c8e02a1f765675c9e8f1ebe7b5c36db6f3c6a9daab2dbbd362f516c45d7
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - libblas
+    - fftw
+    - libpng
+    - zlib
+    - bzip2
+    - libtiff
+    - suitesparse
+  run:
+
+test:
+  commands:
+    - ${PREFIX}/bin/projects/electronMicroscopy/matchSeries --help
+
+about:
+  home: https://github.com/berkels/match-series
+  license: CDDL
+  license_family: OTHER
```

The SPDX identifier has a version, either `CDDL-1.0` or `CDDL-1.1`: https://spdx.org/licenses/

You can then remove `license_family`
```suggestion
  license: CDDL-X.Y
```